docs/source/compression/compression_evaluator.rst

```diff
@@ -0,0 +1,231 @@
+Compression Evaluator Creation
+==============================
+
+The evaluator is used to package the training and evaluation process for models with similar topologies.
+To explain why NNI needs an evaluator, let's first look at the general process of model compression in NNI.
+
+In model pruning, some algorithms need to prune according to some intermediate variables (gradients, activations, etc.) generated during the training process,
+and some algorithms need to gradually increase or adjust the sparsity of different layers during the training process,
+or adjust the pruning strategy according to the performance changes of the model during the pruning process.
+
+In model quantization, NNI has quantization-aware training algorithm,
+it can adjust the scale and zero point required for model quantization from time to time during the training process,
+and may achieve a better performance compare to post-training quantization.
+
+For the support of the above algorithms, NNI previously provided APIs like ``trainer``, ``traced_optimizer``, ``criterion``, ``finetuner``.
```

Comment:
We might put the introduction of legacy APIs in a note. First-come users do not need to know this.

Reply:
good suggestion